### PR TITLE
Newly added Keras Layers python side

### PR DESCRIPTION
**Pull request by hkvision**
## What changes were proposed in this pull request?

upsampling1d, upsampling2d, sepconv2d
update for batchnorm NHWC
some fix; add some ut

## How was this patch tested?

Jenkins

### CODE REVIEW

**Comment by hkvision** [comment]
jenkins passed http://172.168.2.101:8080/view/PR-V/job/BigDL-PR-V/573/
